docs/misc/vmdkops-admin-cli-spec.md

```diff
@@ -11,7 +11,9 @@ All output from the admin cli defaults to human readable formats. It will be mad
 The majority of testing will be automated. We can ensure that parsing calls the right callbacks with
 the right information by generating representative input and mocking the callbacks to assert that
 the right information is parsed and delivered correctly. Additionally, and specifically for testing
-roles, we can create roles and then test that they act as expected by calling `role get`. Unit
+access control, we can create access controk definition (tenants and privileges) 
+and then test that they act as expected by invoking vmdk_ops commmands on behalf of a fake VM.
```

Comment:
Can suggest using the dummy VM created with vim-cmd and say how that can be created.